app/views/image_templates/edit.html.erb

```diff
@@ -8,6 +8,11 @@
   <hr>
 </div>
 
+<% @image_template.match_options['threshold'] ||= 80 %>
```

Comment:
Instead of instantiating these here, could we just assign in the argument pass?

```ruby
value: @image_template.match_options['threshold'] || 80, max: 100, object: f.object_name, attribute: 'match_options', type: 'threshold', formType: 'Range', helpText: t('image_templates.threshold.help')
```